cypress/integration/2.4-custom_js_css.spec.js

```diff
@@ -0,0 +1,44 @@
+import { testSetup, updateRetry } from '../support/common/support.js'
+import { testCustomJSAndCSS } from '../support/outputvalidation.js'
+import selectors from '../support/common/selectors.js'
+import checkoutUiCustomSelectors from '../support/selectors.js'
+
+describe('Testing Custom JS and CSS', () => {
+  // Load test setup
+  testSetup()
+  const { productName, postalCode, prefix } = testCustomJSAndCSS
+
+  it(`In ${prefix} - Adding Product to Cart`, updateRetry(3), () => {
+    // Search the product
+    cy.searchProduct(productName)
+    // Add product to cart
+    cy.addProduct(productName)
+  })
+
+  it(`In ${prefix} Updating product quantity to 2`, updateRetry(3), () => {
```

Comment:
It's really necessary update quantity again? I saw it on the previous test.

Reply:
I will remove it in next commit